plugins/module_utils/cmci.py

```diff
@@ -152,6 +152,33 @@ def is_alphanumeric(value):
     return re.match('^([A-Za-z0-9]{1,100})$', value, flags=0)
 
 
+def read_node(node):
```

Comment:
Looks like this is used both by reading errors and by reading records, so we should make sure we don't have any behaviour in here that's specific to reading feedback nodes.  It's good to re-use the code, but in this case I think we need to tweak it a bit.  Might be good to extract that `(k[1:]v)` tuple into a separate function that could be used in both places.

Reply:
pushed a new commit to address this